### ISSUE TITLE
Installing artifactory startup issue

### ISSUE THREAD

**Issue by vamsikishore2005** [QT-DevOps/DevOpsIssues#8]
After Installing jfrog Not able to brning up service on Ubuntu machine getting below issue.

I am able to check the artifactory .

root@ip-172-31-3-232:/home/ubuntu/artifactory-oss-6.0.1/bin# ./artifactoryctl check
Checking arguments to Artifactory:
ARTIFACTORY_HOME     =  /home/ubuntu/artifactory-oss-6.0.1
ARTIFACTORY_USER     =  artifactory
TOMCAT_HOME          =  /home/ubuntu/artifactory-oss-6.0.1/tomcat
ARTIFACTORY_PID      =  /home/ubuntu/artifactory-oss-6.0.1/run/artifactory.pid
JAVA_HOME            =  /usr/lib/jvm/java-8-oracle/
JAVA_OPTIONS         =  -server -Xms512m -Xmx2g -Xss256k -XX:+UseG1GC -XX:OnOutOfMemoryError="kill -9 %p" -Djruby.compile.invokedynamic=false -Dfile.encoding=UTF8 -Dartdist=zip -Dorg.apache.tomcat.util.buf.UDecoder.ALLOW_ENCODED_SLASH=true -Djava.security.egd=file:/dev/./urandom

root@ip-172-31-3-232:/home/ubuntu/artifactory-oss-6.0.1/bin# ./artifactoryctl start
found java executable in JAVA_HOME
Starting Artifactory tomcat as user artifactory...
/home/ubuntu/artifactory-oss-6.0.1/tomcat/bin/startup.sh: 1: /home/ubuntu/artifactory-oss-6.0.1/tomcat/bin/startup.sh: uname: not found
/home/ubuntu/artifactory-oss-6.0.1/tomcat/bin/startup.sh: 1: /home/ubuntu/artifactory-oss-6.0.1/tomcat/bin/startup.sh: dirname: not found
Cannot find /catalina.sh
The file is absent or does not have execute permission
This file is needed to run this program

** ERROR: Artifactory Tomcat server did not start. Please check the logs
But logs have not generated in tomcat logs folder

**Comment by shaikkhajaibrahim** [comment]
Can you list all the steps you have followed before reaching this stage

**Comment by vamsikishore2005** [comment]
First copied 
1) jfrog-artifactory-oss-6.0.1.zip on to  Ubuntu server.
2) Unzipped and installed jfrog using Executed below command from artifactory bin location
./artifactory.sh
3) Then I have set JAVA_Home variable.
4) Tried to  check ./artifactoryctl check it completed succesfully.
5) When I tried to start this command  ./artifactoryctl start I am getting below issue .

root@ip-172-31-3-232:/home/ubuntu/artifactory-oss-6.0.1/bin# ./artifactoryctl start
found java executable in JAVA_HOME
Starting Artifactory tomcat as user artifactory...
/home/ubuntu/artifactory-oss-6.0.1/tomcat/bin/startup.sh: 1: /home/ubuntu/artifactory-oss-6.0.1/tomcat/bin/startup.sh: uname: not found
/home/ubuntu/artifactory-oss-6.0.1/tomcat/bin/startup.sh: 1: /home/ubuntu/artifactory-oss-6.0.1/tomcat/bin/startup.sh: dirname: not found
Cannot find /catalina.sh
The file is absent or does not have execute permission.

But files are presenting in that location and it has rewrite executable permission
